.buildkite/pipeline.yml

```diff
@@ -6,6 +6,35 @@ env:
 
 # This is the default pipeline – it will build and test the app
 steps:
+  - group: ":eyeglasses: Reader Builds"
+    steps:
+      - label: ":eyeglasses: Reader ASC Demo Build"
+        key: build_asc_reader
+        command: .buildkite/commands/release-build-reader.sh
+        plugins: [$CI_TOOLKIT_PLUGIN]
+        artifact_paths:
+          - Artifacts/*.zip
+          - Artifacts/*.ipa
+        notify:
+          - github_commit_status:
+              context: "Reader ASC Demo Build"
+
+      - block: Distribute Reader via TestFlight
+        prompt: Distribute the Reader build via TestFlight?
+        key: testflight_triggered_reader
+        # Block steps have implicit dependency on the steps that come before them.
+        # See https://buildkite.com/docs/pipelines/trigger-step
+        #
+        # Make it depend on nothing so we don't have to wait for previous steps to finish before the deployment can start.
+        depends_on: ~
+
+      - label: ":eyeglasses: Reader one-off TestFlight Upload"
+        depends_on: [build_asc_reader, testflight_triggered_reader]
```

Comment:
I was expecting `testflight_triggered_reader` to suffice because of the implicit before-after dependency in `block`, but that's not what happened in https://buildkite.com/automattic/wordpress-ios/builds/27273/steps?sid=019624d9-64df-4b64-89da-927be7ecb550

While it's possible I made a mistake elsewhere, this verbose explicit syntax works so I think we can keep it for now.